sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/PropertyConfigurator.java

```diff
@@ -151,4 +159,39 @@ private String getAppLocale() {
     public void setAppLocale(String appLocale) {
         mAppLocale = appLocale;
     }
+
+    /**
+     * Add or overwrite the given key for the common event properties. Properties will be inherited
+     * by children of this transmission target.
+     *
+     * @param key   The property key.
+     * @param value The property value.
+     */
+    @SuppressWarnings("WeakerAccess")
+    public synchronized void setEventProperty(String key, String value) {
+        mEventProperties.put(key, value);
+    }
+
+    /**
+     * Removes the given key from the common event properties.
+     *
+     * @param key The property key to be removed.
+     */
+    @SuppressWarnings("WeakerAccess")
+    public synchronized void removeEventProperty(String key) {
+        mEventProperties.remove(key);
+    }
+
+    /**
```

Comment:
This is not a javadoc. Use `/* */` comment style or adding a full javadoc with discussion